pkg/indexstorage/redis/redis_test.go

```diff
@@ -26,14 +26,14 @@ import (
 )
 
```

Comment:
Can we add a test to confirm read/write works with multiple values?

Reply:
The e2e tests already cover this